src/bower.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "server",
+  "main": "index.html",
+  "dependencies": {
+    "polymer": "Polymer/polymer#^2.0.0",
+    "paper-button": "^2.0.0"
```

Comment:
How do all the components come in? Is this going to have all the elements eventually?

Reply:
Yeah, I was going to start with installing the iron-, paper-, and gold-elements.